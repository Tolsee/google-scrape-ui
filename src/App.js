// @flow
import React from 'react';
import { useUser } from 'context/userContext';
import Authenticated from 'components/Authenticated';
import UnAuthenticated from 'components/UnAuthenticated';

export default function App() {
  const user = useUser();

  return user ? <Authenticated /> : <UnAuthenticated />;
}

