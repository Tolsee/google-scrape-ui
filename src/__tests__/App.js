import React from 'react';
import { mount } from 'enzyme';

import App from '../App';

describe('App component', () => {
  it('should render app component', () => {
    const component = mount(<App />);

    expect(component).toMatchSnapshot();
  });
});
