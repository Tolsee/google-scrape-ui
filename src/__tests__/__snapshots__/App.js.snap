// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render app component 1`] = `
<App>
  <div>
    Home
  </div>
</App>
`;
